testsuites/syncgateway/functional/tests/test_xattrs.py

```diff
@@ -2518,3 +2527,123 @@ def update_prop():
         if(doc_id not in channel_view_query_string or time.time() - start > timeout):
                 break
     assert doc_id not in channel_view_query_string, "doc id exists in chanel view query after compaction"
+
+
+@pytest.mark.syncgateway
+@pytest.mark.xattrs
+@pytest.mark.changes
+@pytest.mark.session
+@pytest.mark.parametrize(
+    'sg_conf_name, number_docs_per_client, number_updates_per_doc_per_client',
+    [
+        ('custom_sync/sync_gateway_custom_sync_require_roles', 10, 10),
+    ]
+)
+def test_stats_logging_import_count(params_from_base_test_setup,
```

Comment:
if this test is only valid for sg 2.5+, do add skip statement for below versions

Reply:
we can run this test for version below 2.5. It has flow which covers error scenario which is not covered yet in existing SGW test flows